projects/modular_llm/train_experts.py

```diff
@@ -19,6 +20,26 @@
 from mttl.utils import generate_random_string, rank_zero_only_and_wait, remote_login
 
 
+def setup_profiler(args: ExpertConfig):
```

Comment:
can we put this in utils ? @matheper I know you want single use code to not be in utils but I think this could be useful somewhere else in the future